coinjoin/coinjoin.go

```diff
@@ -202,11 +202,17 @@ func (t *Tx) ValidateUnmixed(unmixed []byte) error {
 	if other.Version != t.txVersion {
 		return errors.New("coinjoin: different tx versions")
 	}
+	if len(other.TxOut) == 0 {
```

Comment:
we can't do this, the client may have decided to not include an unmixed change output to pay their fee or to exclude it entirely because it is dust.